src/plugins/dashboard/public/dashboard_app/top_nav/dashboard_top_nav.tsx

```diff
@@ -254,7 +220,45 @@ export function DashboardTopNav({ embedSettings, redirectTo }: DashboardTopNavPr
         ref={dashboardTitleRef}
         tabIndex={-1}
       >{`${getDashboardBreadcrumb()} - ${dashboardTitle}`}</h1>
-      <TopNavMenu {...getNavBarProps()} />
+      <TopNavMenu
```

Comment:
Rather than calling and spreading the result of `getNavBarProps`, which goes against our usual pattern for grabbing props, I talked with @ThomThomson and we decided it was best to just put the props in directly - this makes the logic for determining the props much more clear. 

This has **nothing** to do with the bug, it was just some code cleanup :+1: